numpy/core/src/umath/ufunc_object.c

```diff
@@ -5297,6 +5317,16 @@ ufunc_repr(PyUFuncObject *ufunc)
     return PyUString_FromFormat("<ufunc '%s'>", ufunc->name);
 }
 
+static int
+ufunc_traverse(PyObject *self, visitproc visit, void *arg)
```

Comment:
In other function, we use `PyUFuncObject` in the function signature directly, to avoid the need for the cast below

Reply:
adopting